templates/commands/render/action_include_test.go

```diff
@@ -83,7 +83,7 @@ func TestActionInclude(t *testing.T) {
 					},
 				},
 			},
-			wantErr: `path "../file.txt" must not contain ".."`,
+			wantErr: fmt.Sprintf("path %q must not contain \"..\"", filepath.FromSlash("../file.txt")),
```

Comment:
When a string has quotes in in, the convention is to use a raw string with backticks so you don't have to escape the quotes.

So rather than this: 

```
"path %q must not contain \"..\""
```

one could do this:

```
`path %q must not contain ".."`
```

Reply:
Changed to backticks.